.github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md

```diff
@@ -0,0 +1,54 @@
+---
+name:  Dependencies Issue
+about: Use this template for all Dependencies Issues, even those automatically generated
+labels: dependencies
+---
+
+# Description
+- Information about the dependency and its role in the project
+- Current version - what are we running
+- Target Version - what are we changing it to
+- DependaBot PR link if available
+
+## Features and fixes available
+- List of important bug fixes, features or security patches
+- List of the important changes in this version that could affect the workflow
+
+## Additional information
+- Additional information about the version update.
+- Guideline that can be followed for the process
+
+## QA Needs
+< Indicate whether this item needs QA testing. For those that do, indicate any special setup or tool set the QA team might need. >
+
+## Acceptance Criteria
+< list of behaviors that must be present in order to satisfy the required functionality described above. >
+- [ ] Dependency updated in the build
+- [ ] Unit tests passing
+
+## Announcement Message
+devchat: < Dependency name > has been updated to <dependency target version> See <link to new features or this Issue > for available features and fixes
+
+## Progress
+### Ready for Planning
+- [ ] Story and Business Value complete
+- [ ] Initial Acceptance Criteria Defined
+- [ ] The person who will Approve story identified
+- [ ] Proposed Solution provided
+
+### Ready for Review
+- [ ] Testing implemented and passed
+
+### Ready for Pre-Production QA
+- [ ] Issue scored
+- [ ] Special testing instructions noted
+- [ ] Acceptance criteria met - boxes checked
+- [ ] Announcement message created
+
+### Ready for SM Approval
+- [ ] Acceptance Test Results included in Comment
+
+### Ready for PO Approval
+- [ ] Announcement message approved
```

Comment:
shouldn't this bit be much sooner in the process? otherwise there would be a large gap in between announcement and deployment of code changes

Reply:
hmm, somehow lost a whole bunch of comments... What I tried to say previously is that at the moment, I am limited as I do my thing in my pipeline. However, soon the new process will have me doing my approval BEFORE things go to Deploy which will address your stated problem. Now there is another one. I think the current process is to announce only after the Issue is CLOSED which is waaay after the rollout. I think announcement should be made after successful and verified deployment because that's when they see the stuff. If we have to de-announce it because FP rejects the Issue, then so be it. With the new process we can have announcement rolled into the Deployment process. For now, can we leave this?

Reply:
As far as I know everyone does it after deploy and test but 🤷